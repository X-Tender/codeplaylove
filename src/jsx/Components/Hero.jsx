import React from 'react';
import Title from 'Components/Title';

const Hero = () => (
	<section className="hero full-width">
		<video
			autoPlay
			className="hero__video"
			loop
			muted
			poster="assets/video/poster.jpg"
			src="assets/video/codeplaylove.mp4"
		/>
		<Title />
	</section>
);

export default Hero;
