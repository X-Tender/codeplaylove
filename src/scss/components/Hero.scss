.hero {
	position: relative;
	height: 100vh;
	overflow: hidden;
	grid-column: page;
	background: url('../video/poster.jpg');

	&__video {
		background: url('../video/poster.jpg');
		background-size: cover;
		object-fit: cover;
		width: 100%;
		height: 100%;
	}
}
