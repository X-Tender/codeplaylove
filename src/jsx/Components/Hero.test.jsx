import React from 'react';
import { shallow } from 'enzyme';
import Hero from './Hero';

describe('Hero', () => {
	const comp = shallow(<Hero />);

	it('renders correctly', () => {
		expect(comp).toMatchSnapshot();
	});
});
