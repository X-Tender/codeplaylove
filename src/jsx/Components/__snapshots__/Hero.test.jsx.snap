// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<section
  className="hero full-width"
>
  <video
    autoPlay={true}
    className="hero__video"
    loop={true}
    muted={true}
    poster="assets/video/poster.jpg"
    src="assets/video/codeplaylove.mp4"
  />
  <Title />
</section>
`;
